content/en/docs/languages/js/propagation.md

```diff
@@ -62,9 +268,51 @@ const { traceparent, tracestate } = output;
 // across services.
 ```
 
+{{% /tab %}} {{< /tabpane >}}
+
 On the receiving service, you'll need to extract `context` (for example, from
 parsed HTTP headers) and then set them as the current trace context.
 
+{{< tabpane text=true >}} {{% tab TypeScript %}}
+
+```typescript
+// Receiving service
+import { Context, propagation, trace, Span } from '@opentelemetry/api';
+
+// Define an interface for the input object that includes 'traceparent' & 'tracestate'.
+interface Carrier {
+  traceparent?: string;
+  tracestate?: string;
+}
+
+// Assume "input" is an object with 'traceparent' & 'tracestate' keys.
+const input: Carrier = {};
+
+// Extracts the 'traceparent' and 'tracestate' data into a context object.
+//
+// You can then treat this context as the active context for your
+// traces.
+let activeContext: Context = propagation.extract(
+  trace.getTracerProvider().getActiveContext(),
```

Comment:
Could `context.active()` -- as is used in the JS tab below -- be used here?

Reply:
```suggestion
  context.active(),
```

Reply:
Yup.